### DISCUSSION TITLE
How do I match IOC using SO 2.3.10

### DISCUSSION THREAD

**Discussion by Blason** [Security-Onion-Solutions/securityonion#2327] [Unsupported Versions]
Hi Team,

Wondering if there is any way to match IOCs in SO 2.3.10? I do have certain IP addresses and domains how do I match using logstash or is there any other alternate way?

**Comment by dougburks** [comment]
One option would be Zeek Intel:
https://docs.securityonion.net/en/2.3/zeek.html#intel

**Comment by weslambert** [comment]
You could also use TheHive for lookups for observables included with cases pushed from the Alerts interface or Kibana.  Last, you might consider doing something within the pipeline (keep in mind the potential performance implications), with something similar to the following: https://github.com/weslambert/misp2elastic
